### PR TITLE
`treehouses system`

### LINKED ISSUE #1480: `treehouses system` ?
```
treehouses system cpu
treehouses system frequency
treehouses system ram
treehouses system disk
...
```
integration with `treehouses temperature` and `treehouses memory`?

### PR DESCRIPTION
**Pull request by darthnoward**
fixes #1480 
displays real time system information corresponding to arguments
```
root@treehouses:~/cli# ./cli.sh system 

CPU:            4.3% @ 1400MHz
Memory:         360M/926M, 38% used
Disk storage:   4.5G/15G, 30% used
Volt:           1.3875V
Temperature:    47.8°C (118.9°F)
```
with some extent of detecting repeating or false arguments 
```
root@treehouses:~/cli# ./cli.sh system nope volt cpu cpu

CPU:            9.7% @ 1400MHz
Volt:           1.3875V

Error: nope is not an option
```

### CODE REVIEW

**Comment by darthnoward** [comment]
Adjustments proposed in codeclimate are going to affect the proper function of the script. 
Doing a check through loop will make the code looks too chunky
Perhaps anyone got other suggestions?

File: tests/system.bats [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    run "${clicmd}" system all
+    assert_success && assert_output -p 'CPU'
+}
+
+@test "$clinom system volt" {
+    run "${clicmd}" system volt
+    assert_success && assert_output -p 'Volt'
+}
+
+@test "$clinom system disk temperatrue" {
```
Comment (law-lin) [inline tests/system.bats:19]: "temperature" is spelled incorrectly here

File: _treehouses [hunk 2]
```diff
@@ -363,6 +363,13 @@ treehouses wifihidden
 treehouses wifistatus
 treehouses convert
 treehouses resolution
+treehouses system
+treehouses system all
+treehouses system cpu
+treehouses system ram
+treehouses system disk
+treehouses system volt
+treehouses system temperature
```
Comment (law-lin) [inline _treehouses:372]: Let's change this to alphabetical order

File: modules/system.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  echo
+  echo "  Listing real time system information corresponding to arguments"
+  echo
+  echo "  Examples:"
+  echo "    $BASENAME system"
+  echo
+  echo "    $BASENAME volt"
+  echo
+  echo "    $BASENAME disk temperature cpu"
+  echo
```
Comment (law-lin) [inline modules/system.sh:96]: Some descriptions could be added here to explain what these examples should do

File: _treehouses (hunk 2 above)
Comment (darthnoward) [reply _treehouses:372]: thanks for the find!

**Pull request merged**